terminus-ui/select/src/select-animations.ts

```diff
@@ -31,12 +31,12 @@ export const tsSelectAnimations: {
     })),
     state('showing', style({
       opacity: 1,
-      minWidth: 'calc(100% + 32px)', // 32px = 2 * 16px padding
+      minWidth: 'calc(100% + 32px)',
       transform: 'scaleY(1)',
     })),
     state('showing-multiple', style({
       opacity: 1,
-      minWidth: 'calc(100% + 64px)', // 64px = 48px padding on the left + 16px padding on the right
+      minWidth: 'calc(100% + 24px)',
```

Comment:
curious... where does this 24 comes from?

Reply:
I believe it's a combination of 16px and 8px which is 1em + padding. The code was originally copied from material, but that math in the old comment didn't match our new layout